test_typing_inspect.py

```diff
@@ -44,27 +74,35 @@ def test_generic(self):
         T = TypeVar('T')
         samples = [Generic, Generic[T], Iterable[int], Mapping,
                    MutableMapping[T, List[int]], Sequence[Union[str, bytes]]]
-        nonsamples = [int, Union[int, str], Union[int, T], ClassVar[List[int]],
-                      Callable[..., T], ClassVar, Optional, bytes, list]
+        nonsamples = [int, Union[int, str], Union[int, T]]
+        nonsamples += CLASSVAR_GENERIC
+        nonsamples += [Callable[..., T], Optional, bytes, list]
```

Comment:
Add these to the initial value of `nosamples`.

Reply:
ok